packages/connectivity/src/scp-cf/destination/destination-cache.spec.ts

```diff
@@ -296,15 +287,21 @@ describe('destination cache', () => {
 
     const destName = destinationOne.name!;
 
-    it('disables the cache by default', async () => {
+    it('uses the cache by default', async () => {
       await destinationCache.cacheRetrievedDestination(
-        { user_id: 'user', zid: 'tenant' },
+        {
```

Comment:
[q] Why not use the `providerUserToken` here?

Reply:
Done.